clang/lib/Driver/ToolChains/CommonArgs.cpp

```diff
@@ -2753,6 +2753,33 @@ void tools::addMachineOutlinerArgs(const Driver &D,
       addArg(Twine("-enable-machine-outliner=never"));
     }
   }
+
+  auto *CodeGenDataGenArg =
+      Args.getLastArg(options::OPT_fcodegen_data_generate,
```

Comment:
Since `fcodegen_data_generate` is now an alias of `fcodegen_data_generate_EQ`, do we need to check for both here? I'd expect checking for the `_EQ` variant should be enough even if the command-line contains the non-`_EQ` one. Same for the other option.

Reply:
Thanks for the suggestion! I now use `_EQ` flag only for various checks.